frontend/src/components/UploadImageCard.tsx

```diff
@@ -51,7 +47,7 @@ function UploadImagePage(): JSX.Element {
       });
       const data = response.data;
       setImageUrl(data.image);
-      setTimeout(() => setIsLoading(false), 3500);
+      setTimeout(() => setIsLoading(false), 500);
```

Comment:
Really wanted to remove this timeout, but I want to display the loading state properly, and also want loading to not be over 1s.

If we using this in production, I probably would just remove this `setTimeOut`